cilium-cli/connectivity/builder/manifests/client-egress-standalone-dns-proxy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: client-egress-standalone-dns-proxy
+spec:
+  endpointSelector:
+    matchLabels:
+      kind: client
+  egress:
+  - toPorts:
+    - ports:
+      - port: "53"
+        protocol: UDP
+      - port: "53"
+        protocol: TCP
+      rules:
+        dns:
+        - matchPattern: "one.one.one.one"
+    toEndpoints:
+      - matchExpressions:
+          - { key: 'k8s-app', operator: In, values: [ "kube-dns", "coredns", "node-local-dns", "nodelocaldns" ] }
+          - { key: 'io.kubernetes.pod.namespace', operator: In, values: [ "kube-system" ] }
+          - { key: 'io.cilium.k8s.policy.cluster', operator: In, values: [ "{{.ClusterName}}" ] }
```

Comment:
Note that this won't work with Openshift or some NodeLocalDNS setups (because NLD is not an endpoint). It might be worth copying the rules from `client-egress-only-dns`